eservices-monitoring-exporter/src/main/resources/application.conf

```diff
@@ -0,0 +1,13 @@
+interop-be-eservices-monitoring-exporter {
+  storage {
+    kind      = "S3"
+    bucket    = ${ESERVICE_MONITORING_EXPORT_BUCKET}
+    filename  = ${ESERVICE_MONITORING_EXPORT_FILENAME}
+    asndjson  = "false"
+    asndjson  = {?IS_NDJSON_FORMAT}
+  }
+  read-model {
+    db-name           = ${READ_MODEL_DB_NAME}
+    connection-string = "mongodb://"${READ_MODEL_DB_USER}":"${READ_MODEL_DB_PASSWORD}"@"${READ_MODEL_DB_HOST}":"${READ_MODEL_DB_PORT}"/"${READ_MODEL_DB_NAME}"?replicaSet=rs0&readPreference=secondaryPreferred&retryWrites=false"
+  }
```

Comment:
I forgot one point: can you please add the collections names in the configuration as the other jobs? 🙏